### DISCUSSION TITLE
Welcome to netbox-proxbox Discussions!

### DISCUSSION THREAD

**Discussion by emersonfelipesp** [emersonfelipesp/netbox-proxbox#38] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SQRD-Link** [comment]
Someone has to be first :D

I'm Richard, (though friends call me link) and i'm a junior sysadmin for a webhosting company. Also a profound tinkerer 
I came across this project while trying to run netbox and AWX in my homelab. While netbox would like to be the Source of truth It appears the actual Hypervisor is the real source of truth, right? hence my wanting to try proxbox. 

I would love to contribute since i cannot really understand why this project isn't used more. but i lack any kind of coding skills 
guinea pig however i can be.
